jax/_src/scipy/signal.py

```diff
@@ -33,6 +33,68 @@
 from jax._src.typing import Array, ArrayLike
 from jax._src.util import canonicalize_axis, tuple_delete, tuple_insert
 
+@_wraps(osp_signal.fftconvolve)
+def fftconvolve(in1: Array ,in2: Array, mode: str ="same",axes=None) -> Array:
+  # sanity checks
+  in1 = jnp.asarray(in1)
+  in2 = jnp.asarray(in2)
+  if in1.ndim == in2.ndim == 0:  # scalar inputs
+    return in1 * in2
+  elif in1.ndim != in2.ndim:
+      raise ValueError("in1 and in2 should have the same dimensionality")
+  elif in1.size == 0 or in2.size == 0:  # empty arrays
+        return jnp.array([])
+  # warn current limitations
+  if mode != "same":
+    warnings.warn("current fftconvolve() use only mode=same argument")
```

Comment:
This should be a `NotImplementedError` rather than a warning

Reply:
done